webapp/tests/helpers/client-test-helper.jsx

```diff
@@ -0,0 +1,201 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import jqd from 'jquery-deferred';
+import {Client4} from 'mattermost-redux/client';
+
+import WebSocketClient from 'client/websocket_client.jsx';
+
+var HEADER_TOKEN = 'token';
+
+class TestHelperClass {
```

Comment:
I don't know about copying in all these helpers. I feel like we should only copy the ones we need, or we should import them.